src/Metrics.ts

```diff
@@ -14,10 +14,8 @@ interface IBridgeGauges {
 
 export class Metrics {
     public static init(bridge: Bridge) {
-        this.metrics = new PrometheusMetrics();
-        this.metrics.registerMatrixSdkMetrics();
+        this.metrics = bridge.getPrometheusMetrics()
```

Comment:
ChristianLint: :upside_down_face: 
```suggestion
        this.metrics = bridge.getPrometheusMetrics();
```

Reply:
We need you as a service :D